content/_index.en.md

```diff
@@ -75,6 +75,13 @@ A **declaration** also contains some metadata on the **service** on which the **
 
 - - -
 
+### How it works
```

Comment:
I am not sure about adding this section in the documentation. We tend to make the assumption that the documentation should complement the public website rather than duplicate it.

If you find it important to repeat this piece of information, I would prefer that we add it in the beginning, right after “Motivation”, as it is a very high level overview of the process.

If you added this part just for the sake of giving a target to the link on line 111, I suggest to instead either remove the link or point it at the homepage :)

Reply:
Yeah
I had added it to give a target to the link due to the lack of a better option but I agree with your idea of linking it to the main website. I had not thought about that.